src/App.js

```diff
@@ -1,11 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState, useEffect } from "react";
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
+  const [test, setTest] = useState(false);
+
+  useEffect(() => {
+    if (test) {
+      console.log("test > ", test);
+    }
+  }, []);
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has a missing dependency: 'test'. Either include it or remove the dependency array.](https://app.codacy.com/gh/wecode-bootcamp-korea/test-repository/pullRequest?prid=10030662)
```suggestion
  }, [test]);
```